widgetssdk/src/main/java/com/glia/widgets/callvisualizer/EndScreenSharingView.kt

```diff
@@ -0,0 +1,68 @@
+package com.glia.widgets.callvisualizer
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.glia.widgets.R
+import com.glia.widgets.databinding.EndScreenSharingViewBinding
+import com.google.android.material.button.MaterialButton
+import com.google.android.material.theme.overlay.MaterialThemeOverlay
+
+class EndScreenSharingView (
+    context: Context,
+    attrs: AttributeSet?,
+    defStyleAttr: Int,
+    defStyleRes: Int)
+    : ConstraintLayout(
+    MaterialThemeOverlay.wrap(context, attrs, defStyleAttr, defStyleRes),
+    attrs,
+    defStyleAttr,
+    defStyleRes
+), EndScreenSharingContract.View {
+
+    var onFinishListener: OnFinishListener? = null
+    private var controller: EndScreenSharingContract.Controller? = null
+
+    private val binding: EndScreenSharingViewBinding by lazy {
+        EndScreenSharingViewBinding.inflate(LayoutInflater.from(this.context), this)
```

Comment:
there is extension function `layoutInflater` you can use instead of `LayoutInflater.from(this.context)`